nft_ingester/src/tasks/mod.rs

```diff
@@ -158,7 +158,8 @@ impl TaskManager {
                             .less_or_equal(Expr::col(tasks::Column::MaxAttempts)),
                     ),
             )
-            .order_by_desc(tasks::Column::CreatedAt)
+            .order_by(tasks::Column::Attempts, Order::Asc)
+            .order_by(tasks::Column::CreatedAt, Order::Desc)
```

Comment:
Shouldn't this order createdAt by descending? We want to treat the tasks like a queue.

Reply:
I belive these comments are addressed? It processes newer records as they come in but will downgrade older ones.